packages/virtual-core/src/index.ts

```diff
@@ -34,6 +34,8 @@ export interface VirtualItem {
   start: number
   end: number
   size: number
+  top: number
+  left: number
```

Comment:
Until now the naming for the properties was orientation-agnostic:

- `start` would either refer to the element's x or y coordinate depending on `options.horizontal`
- `size` would either refer to the element's width or height
- `end` is `start` + `size` so it would either refer to x2 or y2

I can see how it makes the code simpler in the library in some places but I feel like it can be confusing for the end user (e.g `translateY(${row.start})`).

Furthermore, with this PR elements are positioned on both axis and I don't know what name would make sense for the opposite axis. I'll think about it but maybe it's fine to have `start` as well as `top` and `left`.

Reply:
Turns out there's no need for top/left so I removed them.